server/server/src/scheduled_tasks.rs

```diff
@@ -2,16 +2,19 @@ use service::service_provider::ServiceProvider;
 use std::{sync::Arc, time::Duration};
 use tokio::task::JoinHandle;
 
-static TASK_INTERVAL: Duration = Duration::from_secs(10);
+pub fn spawn_scheduled_task_runner(
+    service_provider: Arc<ServiceProvider>,
+    interval_secs: u64,
+) -> JoinHandle<()> {
+    let period: Duration = Duration::from_secs(interval_secs);
```

Comment:
Why switch between period and interval?
It looks fine but wondering if we should stick to one name?

Reply:
Oh haha was just using the naming from the Duration package inputs, but I reckon I could just do it all in one go 😅